pkg/credentialschema/testdata/issuer_metadata.json

```diff
@@ -1,21 +1,26 @@
 {
-  "authorization_endpoint":"https://server.example.com/oidc/authorize",
+  "credential_issuer":"https://server.example.com",
+  "authorization_server":"https://server.example.com/oidc/authorize",
   "credential_endpoint":"https://server.example.com/oidc/credential",
-  "credential_issuer":{
-    "display":[
-      {
-        "locale":"en-US",
-        "name":"Example University"
-      },
-      {
-        "name":"サンプル大学",
-        "locale":"jp-JA"
-      }
-    ]
-  },
-  "credentials_supported":{
-    "university_degree":{
-      "claims":{
+  "display":[
+    {
+      "locale":"en-US",
+      "name":"Example University"
+    },
+    {
+      "name":"サンプル大学",
+      "locale":"jp-JA"
```

Comment:
Shouldn't `locale` come as a second key?

Reply:
The order doesn't matter, but it would look nicer if it was consistent. Next time I modify that file I'll update it